app/server/middleware/apiMiddleware.ts

```diff
@@ -1,83 +0,0 @@
-import { Request, Response } from "express";
-import fetch from "node-fetch";
-import { parse } from "url";
-import { X_GU_ID_FORWARDED_SCOPE } from "../../shared/identity";
-import { MDA_TEST_USER_HEADER } from "../../shared/productResponse";
-import { conf } from "../config";
-import { log } from "../log";
-import {
-  augmentRedirectURL,
-  getCookiesOrEmptyString
-} from "./identityMiddleware";
-
-export type BodyHandler = (res: Response, body: string) => void;
-
-export const straightThroughBodyHandler: BodyHandler = (res, body) =>
-  res.send(body);
-
-export const proxyApiHandler = (
```

Comment:
this file is showing up as a delete, when really its a rename and move to `app/server/apiProxy.ts` - since this isn't strictly middleware